codegens/curl/lib/index.js

```diff
@@ -186,6 +186,15 @@ self = module.exports = {
         description: 'Set a character used to mark the continuation of a statement on the next line ' +
           '(generally, \\ for OSX/Linux, ^ for Windows)'
       },
+      {
+        name: 'Quote Type',
+        id: 'quoteType',
+        availableOptions: ['\'', '"'],
```

Comment:
availableOptions should be ['singleQuote', 'doubleQuote'] as mentioned by @abhijitkane above.

Reply:
Fixed